iceoryx_posh/test/moduletests/test_mepoo_handler.cpp

```diff
@@ -349,3 +349,42 @@ TEST_F(MemoryManager_test, addMemPoolWithChunkCountZeroShouldFail)
     mempoolconf.addMemPool({32, 0});
     EXPECT_DEATH({ sut->configureMemoryManager(mempoolconf, allocator, allocator); }, ".*");
 }
+
+TEST_F(MemoryManager_test, MempoolIndexOutOfRange)
```

Comment:
Please rename the test into the format: PerformSomethingAndExpectBlubb 
In this case `FillingMempoolCompletelyAndAccessingOutOfRangeAccessProvidesEmptyMemPoolInfo`
I know its long but then you clearly state your test intention and we know how we should review the test.